includes/abstracts/abstract-wc-legacy-product.php

```diff
@@ -375,4 +375,18 @@ public function get_cross_sells() {
 		_deprecated_function( 'WC_Product::get_cross_sells', '2.7', 'WC_Product::get_cross_sell_ids' );
 		return apply_filters( 'woocommerce_product_crosssell_ids', (array) maybe_unserialize( $this->crosssell_ids ), $this );
 	}
+
+	/**
+	 * Check if variable product has default attributes set.
+	 *
+	 * @access public
```

Comment:
I've been removing these public access docblocks. Should they be left if public or defined? https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/

Needs a deprecated docblock too.